src/pages/TelaCadastroDadosAcesso.jsx

```diff
@@ -0,0 +1,195 @@
+    let [fontsLoaded] = useFonts({
+        'Barlow-Light': require("../assets/fonts/Barlow-Light.ttf"),
+    });
+    if (!fontsLoaded) {
+        return <AppLoading />
+    }
+    function buttonEntrar() {
+        navigate(routes.TELA_MENU)
+
+    }
+
+    return (
+        <SafeAreaView>
+            <View style={styles.div}>
+                <View style={styles.header}>
+                    <Text style={styles.dogtorText}>DOGTOR</Text>
+                    <Text style={{ fontWeight: 'bold', fontSize: 24 }}>Dados de Acesso</Text>
+                    <Text style={{ fontSize: 16 }}>Muito bom conhecer o dono desse pet. Precisamos das suas informações de acesso</Text>
+                </View>
+                <View>
+                    {<Controller
+                        control={control}
+                        name="email"
+                        rules={{ required: true }}
+                        render={({ field: { value, onChange } }) => (
+                            <TextInput
+                                style={{
+                                    height: 51,
+                                    width: 360,
+                                    marginTop: 12,
+                                    borderWidth: 1,
+                                    padding: 10,
+                                    borderRadius: 16,
+                                    borderColor: errors.email && 'red' || !errors.email && '#ACBBC3',
+                                }}
+                                value={value}
+                                onChangeText={onChange}
+                                placeholder="E-mail"
+                            />
+                        )}
+                    />}
+
+                    {<Controller
+                        control={control}
+                        name="telefone"
+                        rules={{ required: true }}
+                        render={({ field: { value, onChange } }) => (
+                            <TextInput
+                                style={{
+                                    height: 51,
+                                    width: 360,
+                                    marginTop: 12,
+                                    borderWidth: 1,
+                                    padding: 10,
+                                    borderRadius: 16,
+                                    borderColor: errors.telefone && 'red' || !errors.telefone && '#ACBBC3',
+                                }}
+                                value={value}
+                                onChangeText={onChange}
+                                placeholder="Telefone"
+                                keyboardType="numeric"
+
+                            />
+                        )}
+                    />}
+                    {<Controller
+                        control={control}
+                        name="Senha"
```

Comment:
aqui acho que da pra colocar em minúsculo para manter o padrão!
```suggestion
                        name="senha"
```